### ISSUE TITLE
Incorrect puzzle format in verify.groovy

### ISSUE THREAD

**Issue by krzyk** [jcabi/jcabi-maven-skin#11]
https://github.com/jcabi/jcabi-maven-skin/blob/d5b3ecf72ad02fb2d16ba6d75200b6aac0defe3b/src/it/basics/verify.groovy#L70  puzzle text should be indented by additional space.
Right now it is causing rultor build to fail (see http://www.rultor.com/t/1562-69751390)

**Comment by dmarkov** [comment]
I'll ask someone to take care of this task soon

**Comment by dmarkov** [comment]
@maurezen please help, the task is yours

**Comment by dmarkov** [comment]
@krzyk thanks for the ticket, your account was topped for 15 mins, payment 50259825

**Comment by maurezen** [comment]
@dmarkov created a pull request here #12

**Comment by maurezen** [comment]
@dmarkov @yegor256 shouldn't there also be my account label on this ticket?

**Comment by maurezen** [comment]
@dmarkov pull request review blocked by #13

**Comment by maurezen** [comment]
@krzyk pull request #12 merged into master. Please close the issue.

**Comment by dmarkov** [comment]
@maurezen **30 mins** sent to your balance (ID `50573301`), many thanks!. +30 added to your rating, current score is: [+705](http://www.netbout.com/b/35102?open=rating)
